woocommerce/woocommerce
# Fix e2e downloading multiple puppeteer

Followup PR to #27113 

### All Submissions:


### Changes proposed in this Pull Request:

Doing as a separate PR to allow #27113 to be reviewed & merged. #27113 adds `@automattic/puppeteer` to the e2e-environment. 

- `@automattic/puppeteer` uses Puppeteer `^2.0.0` which has `jest ^24.9.0` as a dependency. 
- Our jest version was updated to `^25.1.0` in #27090 
- WC Admin downgraded to `jest ^24.9.0` in https://github.com/woocommerce/woocommerce-admin/pull/4942

I set up a new e2e test environment and saw that each version of `jest` was downloading its own dependent version of Chromium.

Closes # .

### How to test the changes in this Pull Request:

1. Set up a new test install
2. `npm install`
3. Only one copy of Chromium should be downloaded
4. `npm run build:packages`
5. `npm run docker:up`
6. `npm run test:e2e`
7. `npm run docker:down`
8. E2E tests should run to completion

### Other information:


### Changelog entry

> Use consistent puppeteer versions in e2e environment
